react/Icon/Readme.md

```diff
@@ -0,0 +1,31 @@
+Provides an easy way to use SVG icons included in Cozy-UI as well
+as your custom icons.
+
+### Cozy Icons
+
+The icons should be colorable but for that, they need to have no fill
+property. Work is in progress to remove the fill attribute on our icons.
+
+```
+const icons = require('../../src/icons');
+const colors = ['red', 'blue', 'yellow', 'green']
+let i = 0;
+
+<div style={{ background: '#ccc'}}>{Object.keys(icons).map(icon => <span>
+  { icon } : <Icon icon={ icon } color={ colors[i++] } /><br/>
+</span>) }</div>
+```
```

Comment:
Beware of the code sample, it lacks a closing parenthesis in the map function.
And format the code as it could be more readable.

Reply:
no parenthesis lacking here